bin/check_samplesheet.py

```diff
@@ -20,7 +20,7 @@ def print_error(error,line):
 
 
 def check_samplesheet(FileIn,FileOut):
-    HEADER = ['sample', 'fastq', 'barcode', 'genome', 'transcriptome']
+    HEADER = ['sample', 'sample_path', 'barcode', 'genome', 'transcriptome','condition']
```

Comment:
```suggestion
    HEADER = ['sample', 'input_file', 'barcode', 'genome', 'transcriptome','condition']
```

Reply:
`input_file` is a bit more generic and sounds better 👍

Reply:
sample_path has been changed to input_file.